gitsome/config.py

```diff
@@ -99,6 +99,9 @@ class Config(object):
 
     :type verify_ssl: bool
     :param verify_ssl: Determines whether to verify SSL certs.
+
+    :type disable_avatar: bool
```

Comment:
Can we call this enable_avatar to avoid double negatives if it gets set to False?